net-im/signal-desktop-bin/signal-desktop-bin-7.57.0.ebuild

```diff
@@ -0,0 +1,94 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+MY_PN="${PN/-bin/}"
+
+inherit pax-utils unpacker xdg
+
+DESCRIPTION="Allows you to send and receive messages of Signal Messenger on your computer"
+HOMEPAGE="https://signal.org/
+	https://github.com/signalapp/Signal-Desktop"
+SRC_URI="https://updates.signal.org/desktop/apt/pool/s/${MY_PN}/${MY_PN}_${PV}_amd64.deb"
+S="${WORKDIR}"
+
+LICENSE="GPL-3 MIT MIT-with-advertising BSD-1 BSD-2 BSD Apache-2.0 ISC openssl ZLIB APSL-2 icu Artistic-2 LGPL-2.1"
+SLOT="0"
+KEYWORDS="-* amd64"
+RESTRICT="splitdebug"
+
+RDEPEND="
+	>=app-accessibility/at-spi2-core-2.46.0:2
+	dev-libs/expat
+	dev-libs/glib:2
+	dev-libs/nspr
+	dev-libs/nss
+	>=media-fonts/noto-emoji-20231130
+	media-libs/alsa-lib
+	|| (
+		media-libs/libpulse
+		media-sound/apulse
+	)
+	media-libs/mesa[X(+)]
+	net-print/cups
+	sys-apps/dbus
+	virtual/udev
+	x11-libs/cairo
+	x11-libs/gdk-pixbuf:2
+	x11-libs/gtk+:3[X]
+	x11-libs/libdrm
+	x11-libs/libX11
+	x11-libs/libxcb
+	x11-libs/libxkbcommon
+	x11-libs/libXcomposite
+	x11-libs/libXdamage
+	x11-libs/libXext
+	x11-libs/libXfixes
+	x11-libs/libXrandr
+	x11-libs/pango
+"
+
+QA_PREBUILT="
+	opt/Signal/chrome_crashpad_handler
+	opt/Signal/chrome-sandbox
+	opt/Signal/libEGL.so
+	opt/Signal/libGLESv2.so
+	opt/Signal/libffmpeg.so
+	opt/Signal/libvk_swiftshader.so
+	opt/Signal/libvulkan.so.1
+	opt/Signal/resources/app.asar.unpacked/node_modules/*
+	opt/Signal/signal-desktop
+	opt/Signal/swiftshader/libEGL.so
+	opt/Signal/swiftshader/libGLESv2.so"
+
+src_prepare() {
+	default
+	sed -e "s|^Exec=/opt/Signal/signal-desktop|Exec=${MY_PN}|" \
+		-i usr/share/applications/signal-desktop.desktop || die
+	unpack usr/share/doc/signal-desktop/changelog.gz
+}
+
+src_install() {
+	insinto /
+	dodoc changelog
+	doins -r opt
+	insinto /usr/share
+
+	doins -r usr/share/applications
+	doins -r usr/share/icons
+	fperms +x /opt/Signal/signal-desktop /opt/Signal/chrome-sandbox /opt/Signal/chrome_crashpad_handler
+	fperms u+s /opt/Signal/chrome-sandbox
+	pax-mark m opt/Signal/signal-desktop opt/Signal/chrome-sandbox opt/Signal/chrome_crashpad_handler
+
+	newbin - signal-desktop <<- _EOF_
+		#!/bin/sh
+		exec \$(command -pv apulse) ${EPREFIX}/opt/Signal/signal-desktop "\${@}"
```

Comment:
Could this be updated to have the parameter ` --ozone-platform-hint=auto` as hinted in bug 950767? Does it work on both X and wayland?